app/javascript/app/pages/ndc-country/ndc-country-component.jsx

```diff
@@ -3,22 +3,47 @@ import Proptypes from 'prop-types';
 import Header from 'components/header';
 import Intro from 'components/intro';
 import Accordion from 'components/accordion';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Search from 'components/search';
+import cx from 'classnames';
 
-class NDCCountry extends PureComponent {
-  componentWillMount() {
-    const { hasData, match, fetchCountryNDC } = this.props;
-    const { iso } = match.params;
-    if (!hasData && iso) {
-      fetchCountryNDC(iso);
-    }
-  }
+import backIcon from 'assets/icons/back.svg';
+import lightSearch from 'styles/themes/search-light.scss';
+import background from 'assets/backgrounds/home_bg_1';
+import styles from './ndc-country-styles.scss';
 
+class NDCCountry extends PureComponent {
   render() {
-    const { countryName } = this.props;
+    const { country, match } = this.props;
     return (
       <div>
-        <Header>
-          <Intro title={countryName} />
+        <Header image={background}>
+          <div className={cx(styles.doubleFold, styles.header)}>
+            <div className={styles.title}>
+              <Button
+                className={styles.backIcon}
+                color="transparent"
+                link="/ndcs"
+                type="icon"
```

Comment:
I don't like it too much if we need a square, why don't pass a prop called square as a bolean? I think is much easy to understand. What do you think?